support/dynamics/gp/guidelines-generate-eft-files.md

```diff
@@ -410,14 +374,14 @@ Field: EFT Next File Number
 
 For more information, see [Custom data fields for Electronic Funds Transfer (EFT) formats in Microsoft Dynamics GP](custom-data-fields-for-electronic-funds-transfer-formats.md).
 
-Q33: What are some other custom fields commonly requested for EFT?
+Q30: What are some other custom fields commonly requested for EFT?
 
-A33: Information on other custom fields commonly requested for EFT file can be found in [Custom Data Fields for Electronic Funds Transfer (EFT) formats in Microsoft Dynamics GP](custom-data-fields-for-electronic-funds-transfer-formats.md).
+A30: Information on other custom fields commonly requested for EFT file can be found in [Custom Data Fields for Electronic Funds Transfer (EFT) formats in Microsoft Dynamics GP](custom-data-fields-for-electronic-funds-transfer-formats.md).
 
-Q34: Can I get an addenda count in the Detail line to achieve the CTX file format?
+Q31: Can I get an addenda count in the Detail line to achieve the CTX file format?
 
-A34: No, at this time, you aren't able to count/sum until after the lines are listed. So you wouldn't be able to count the addenda lines in the Detail line since they're printed after the Detail line, so you wouldn't be able to achieve the CTX format. Your options would include:
+A31: No, at this time, you aren't able to count/sum until after the lines are listed. So you wouldn't be able to count the addenda lines in the Detail line since they're printed after the Detail line, so you wouldn't be able to achieve the CTX format. Your options would include:
 
 1. Run 'one check per invoice' instead of 'one check per vendor', and hard code the addenda count =1 in the EFT file format.
-1. Ask the bank if they would make an exception or accept any other NACHA format such as the CCD or PPD format. (Many banks have accepted the 'detail + addenda' count in a footer line instead.)
-1. Seek out a third-party product that allows this functionality or 4) a dexterity customization.
+2. Ask the bank if they would make an exception or accept any other NACHA format such as the CCD or PPD format. (Many banks have accepted the 'detail + addenda' count in a footer line instead.)
+3. Seek out a third-party product that allows this functionality or 4) a dexterity customization.
```

Comment:
Hi @theley502, can you help double check the meaning of the "4)" in line 387? I'm not sure whether it's correct. Thanks!

Reply:
looks OK